sound/soc/sof/intel/hda.h

```diff
@@ -343,6 +343,12 @@ struct sof_intel_dsp_desc {
 	struct snd_sof_dsp_ops *ops;
 };
 
+enum sof_hda_tream_type {
+	SOF_HDA_AUDIO_STREAM = 0,	/* normal playback/capture stream */
```

Comment:
"audio" is confusing, we could use this for compressed and probes. Maybe "normal" or just no qualifier.

Reply:
This was "normal" in initial version, I can change it back if we all aligned, this is not so important to me.

Reply:
OK, will change.